modules/api/src/main/java/org/apache/ignite/deployment/UnitStatus.java

```diff
@@ -69,6 +92,7 @@ public static UnitStatusBuilder builder(String id) {
         return new UnitStatusBuilder(id);
     }
 
+    /** {@inheritDoc} */
```

Comment:
I don't think these are necessary. At least there's no rule for having them.
Maybe there's a rule for not having them actually, I didn't check